pinot-transport/src/test/java/com/linkedin/pinot/transport/perf/ScatterGatherPerfClient.java

```diff
@@ -297,39 +295,20 @@ public SimpleScatterGatherRequest getRequest() throws IOException, JSONException
    * @return
    * @throws InterruptedException
    * @throws ExecutionException
-   * @throws IOException
-   * @throws ClassNotFoundException
    */
   private String sendRequestAndGetResponse(SimpleScatterGatherRequest request,
-      final ScatterGatherStats scatterGatherStats) throws InterruptedException,
-      ExecutionException, IOException, ClassNotFoundException {
+      final ScatterGatherStats scatterGatherStats) throws InterruptedException, ExecutionException {
     BrokerMetrics brokerMetrics = new BrokerMetrics(new MetricsRegistry());
-    CompositeFuture<ByteBuf> future = _scatterGather.scatterGather(request, scatterGatherStats,
-        brokerMetrics);
-    ByteBuf b = future.getOne();
-    String r = null;
-    if (null != b) {
-      byte[] b2 = new byte[b.readableBytes()];
-      b.readBytes(b2);
-      r = new String(b2);
-
-    }
-    return r;
-  }
-
-  private static void releaseByteBuf(CompositeFuture<ByteBuf> future) throws Exception {
-    Map<ServerInstance, ByteBuf> bMap = future.get();
-    if (null != bMap) {
-      for (Entry<ServerInstance, ByteBuf> bEntry : bMap.entrySet()) {
-        ByteBuf b = bEntry.getValue();
-        if (null != b) {
-          b.release();
-        }
-      }
+    CompositeFuture<byte[]> future = _scatterGather.scatterGather(request, scatterGatherStats, brokerMetrics);
+    byte[] bytes = future.getOne();
+    if (bytes == null) {
```

Comment:
bytes == null || bytes.length == 0

Reply:
empty byte array could be a valid response